help/en/html/tools/signaling/SignalMasts.shtml

```diff
@@ -8,7 +8,7 @@
   <!-- Copyright Bob Jacobsen 2008 -->
 
   <title>JMRI: Signal Masts documentation</title>
-  <meta name="Author" content="Bob Jacobsen, Egbert Broerse">
+  <meta name="Author" content="Bob Jacobsen, Egbert B79roerse">
```

Comment:
This change seem to have gone wrong. `Egbert B79roerse`

Reply:
Should be resolved in lastest commit.